### ISSUE TITLE
track visits to each collection

### ISSUE THREAD

**Issue by saracarl** [benwbrum/fromthepage#1376]
Even if not logged in -- or if we could embed Google Analytics tags into our site instead that would be even better.

**Comment by bencomp** [comment]
This sounds related to #1747 (which provides a potential solution).

**Comment by benwbrum** [comment]
I agree with @bencomp -- letting project owners embed whatever they want in the footer already works, and makes a lot more sense than hard-coding GA tracking.  Now that the footer is displayed for all projects rather than per collection, it's also less error prone.
